### DISCUSSION TITLE
Subscriptions (with Redis) Firing many times for one publish

### DISCUSSION THREAD

**Discussion by sgentile** [MichalLytek/type-graphql#1009] [Q&A]
The subscriptions are working, however, I'm noticing where I publish once, and get my Subscription is firing off 7-50 times with the same message over and over again.

Is this have something to do with redis not clearing messages from the queue ?   I'm not sure how to troubleshoot since it's more of the internals of @Subscription ?

**Comment by MichalLytek** [comment]
Check how it works with an in-memory PubSub, so you will see if that's an issue with Redis.
